Jellyfin.Server/Startup.cs

```diff
@@ -96,6 +98,15 @@ public void ConfigureServices(IServiceCollection services)
                 })
                 .ConfigurePrimaryHttpMessageHandler(defaultHttpClientHandlerDelegate);
 
+            services.AddHttpClient(NamedClient.DirectIp, c =>
```

Comment:
What's the difference between `Default` and `DirectIp`? Right now they seem to be the same

Reply:
just updated after bond's change.